lib/index.js

```diff
@@ -4718,30 +4719,36 @@ function getOpamFileName(version) {
 function getOpamDownloadUrl(version, filename) {
     return util.format("https://github.com/ocaml/opam/releases/download/%s/%s", version, filename);
 }
-function acquireOpamWindows(version) {
+function acquireOpamWindows(version, custom_repository) {
     return __awaiter(this, void 0, void 0, function () {
-        var downloadPath, error_1, toolPath;
+        var repository, downloadPath, error_1, toolPath;
         return __generator(this, function (_a) {
             switch (_a.label) {
                 case 0:
-                    _a.trys.push([0, 2, , 3]);
-                    return [4 /*yield*/, tc.downloadTool("https://cygwin.com/setup-x86_64.exe")];
+                    repository = custom_repository === ""
+                        ? "https://github.com/fdopen/opam-repository-mingw.git#opam2"
+                        : custom_repository;
+                    _a.label = 1;
                 case 1:
-                    downloadPath = _a.sent();
-                    return [3 /*break*/, 3];
+                    _a.trys.push([1, 3, , 4]);
```

Comment:
Is this correct? `[1, 3, , 4]`?

Reply:
The contents of `lib/` is the output of transpiling the typescript in `src/`